controllers/controllers/networking/cfroute_controller.go

```diff
@@ -165,41 +152,27 @@ func createInvalidRouteStatus(cfRoute *korifiv1alpha1.CFRoute, description, reas
 }
 
 func (r *CFRouteReconciler) setRouteErrorStatusAndReturn(ctx context.Context, cfRoute *korifiv1alpha1.CFRoute, err error, description, reason string) (ctrl.Result, error) {
```

Comment:
This function is a bit weird, since it takes an error argument simply to return it. Also it does not seem to use the context argument. To me it would look better if it was called `setRouteErrorStatus` and did not return an error. Then its usage will be something like:

```
if apierrors.IsNotFound(err) {
	r.setRouteErrorStatus(cfRoute, "CFDomain not found", "InvalidDomainRef")
        return err
}
```

Reply:
Sounds reasonable
+1